### ISSUE TITLE
Missing `CADETS_E5.pkl` for the documented `CADETS_E5 --from_weights` reproduction

### ISSUE THREAD

**Issue by Mengyu-95** [ubc-provenance/orthrus#17]
Hi Orthrus authors,
Thank you for open-sourcing the code and reproduction instructions.
I am trying to reproduce the documented `CADETS_E5` experiment from the README. The README currently includes the command:
PYTHONHASHSEED=0 python src/orthrus.py CADETS_E5 --from_weights --detection.gnn_training.node_out_dim=128 --detection.gnn_training.lr=0.0001 --detection.gnn_training.encoder.graph_attention.dropout=0.1 --graph_construction.build_graphs.time_window_size=1.0

However, in the current public weights/ directory, I can only find:
* CADETS_E3.pkl
* CLEARSCOPE_E3.pkl
* CLEARSCOPE_E5.pkl
* THEIA_E3.pkl
* THEIA_E5.pkl

I cannot find a matching CADETS_E5.pkl. I also tried using a CADETS_E5.pkl obtained from a PIDSMaker/velox-related weights folder, but it appears incompatible with the current ubc-provenance/orthrus codebase: loading it results in state_dict mismatch errors.
Could you provide/upload the correct CADETS_E5.pkl compatible with the current `main` branch?

Thanks a lot for your help.

**Comment by TristanBilot** [comment]
Hi @Mengyu-95,
We don't have access to pretrained weights for CADETS_E5 unfortunately. You should be able to approach the results from the paper after hyperparameter tuning across different `node_hid_dim`, `node_out_dim` and `lr`.
We usually recommend to use PIDSMaker directly for that. Let me know if you have any other question.
